### ISSUE TITLE
[BUG] - btltbl.bin alterations causing glitched text

### ISSUE THREAD

**Issue by SuperfieldCU** [OpenKH/OpenKh#1084]
**OpenKH Build**
24.07.17.0

**OpenKH Program**
Mod Manager

**Describe the Bug**
In KH1FM, when using a mod or combination of mods via Panacea that include BOTH:
- The `btltbl.bin` file in the root of the `kh1` file directory, AND
- The files within `remastered/btltbl.bin` (exact file(s) unknown, but I at least know that problem occurs when just the files `UK_AbilityName.bin`, `UK_ItemHelp.bin`, and `UK_Word.bin` are present);
a lot of system text is glitched and uses seemingly random characters. The areas where this happens includes the command menu, the magic shortcut menu, and most camp menus (items, equipment, abilities, and Sora's customization).

This occurs even when the modded files are unaltered from stock.

**To Reproduce**
1. Install one or more KH1 mods that include `btltbl.bin`, `remastered/btltbl.bin/UK_AbilityName.bin`, `remastered/btltbl.bin/UK_ItemHelp.bin`, and `remastered/btltbl.bin/UK_Word.bin` via Panacea. The files do not need to have been altered from stock.
2. Run KH1 and open a save file.
3. You will see the glitched text in the command menu.

**Expected behavior**
If the files in the mod were unchanged from stock, the command menu and other text should appear identical to how they would if the game was unmodded.

**Screenshots**
![20240720224640_1](https://github.com/user-attachments/assets/4f197caa-7882-4f52-a7d0-79cb6787e7b0)
![20240720224709_1](https://github.com/user-attachments/assets/96f4103a-2925-41e6-9b10-965dbcd553da)
![20240720224721_1](https://github.com/user-attachments/assets/c0c92897-f1f2-4d57-93a9-53333c2c641b)
![20240720224728_1](https://github.com/user-attachments/assets/ebe1a1a9-c2d9-45df-8f16-19eed718b89c)
![20240720224735_1](https://github.com/user-attachments/assets/66bb5fe2-4c55-4357-bce9-49ff1c4d536d)
![20240720230625_1](https://github.com/user-attachments/assets/966f8c36-a4b2-4b95-bccd-d01ccb6b5a65)

**Additional context**
I have used a mod that alters the three reported files in `remastered/btltbl.bin` for some time now with no issues; it was only when I introduced a file that altered `btltbl.bin` that the text suddenly became screwed up. Turning off the former mod and only using the latter also produced no issues. It's only when the two are used together that problems occur.

**Comment by SuperfieldCU** [comment]
No, mods do not need to alter any text to produce the glitched text. In fact, no changes need to be made at all; even if Panacea is loading stock versions of `btltbl.bin` and `remastered/btltbl.bin/UK_AbilityName.bin`, the text will still be glitched. You can try it out for yourself with the attached file; it contains a KH1PCPATCH file that only contains stock versions of those two files.
[BtlTbl Test.zip](https://github.com/user-attachments/files/16939794/BtlTbl.Test.zip)

**Comment by MainMemory** [comment]
Can confirm this is indeed the case, though I haven't the faintest idea why as yet.
